inc/class-command.php

```diff
@@ -285,4 +267,102 @@ protected function get_project_hosts() : array {
 
 		return $hosts;
 	}
+
+	/**
+	 * Get the root directory path for the project.
+	 *
+	 * @return string
+	 */
+	protected function get_root_dir() : string {
+		return dirname( $this->getComposer()->getConfig()->getConfigSource()->getName() );
+	}
+
+	/**
+	 * Get the Local Chassis config from composer.json.
+	 *
+	 * @return array
+	 */
+	protected function get_config() : array {
+		// @codingStandardsIgnoreLine
+		$json = file_get_contents( $this->get_root_dir() . DIRECTORY_SEPARATOR . 'composer.json' );
+		$composer_json = json_decode( $json, true );
+
+		return (array) $composer_json['extra']['altis']['modules']['local-chassis'] ?? [];
+	}
+
+	/**
+	 * Writes the config.local.yaml file with Altis customisations.
+	 *
+	 * @return bool Returns false if the file write fails.
+	 */
+	protected function write_config_file() : bool {
+		// Get project host names from config.
+		$hosts = $this->get_project_hosts();
+
+		// Write the default config.
+		$config = [
+			'machine_name' => $hosts[0],
+			'php' => '7.2',
+			'paths' => [
+				'base' => '..',
+				'wp' => 'wordpress',
+				'content' => 'content',
+			],
+			'hosts' => $hosts,
```

Comment:
Does this need to be set manually now, or will it be set automatically via merge_config?

Reply:
Derp, yeah it will. Although it needs a default value still which the `get_project_hosts()` function provides. Could always set that same default here and force the machine name below.

Reply:
Updated